packages/nextjs/src/server/index.ts

```diff
@@ -260,6 +261,16 @@ export function init(options: NodeOptions): NodeClient | undefined {
             }
           }
 
+          // On Edge Runtime, fetch requests can leak to the Node runtime fetch instrumentation
+          // and end up being sent as transactions to Sentry.
+          // We filter them here based on URL
+          if (event.contexts?.trace?.data?.[SEMANTIC_ATTRIBUTE_SENTRY_OP] === 'http.client') {
+            const url = event.contexts?.trace?.data?.['url'];
+            if (url && isSentryRequestUrl(url, client)) {
+              return null;
+            }
+          }
+
           return event;
         } else {
           return event;
```

Comment:
A bit unrelated to this PR, is there a reason why we are returning the event in an `else` block?
Wouldn't it make more sense to remove the `else` and return the event after the `if` statement?